can/io/blf.py

```diff
@@ -173,6 +182,11 @@ def __init__(
         )
         # Read rest of header
         self.file.read(header[1] - FILE_HEADER_STRUCT.size)
+
+        # APP_TEXT objects are appended when type 65 is encountered
+        # while iterating
+        self.app_text: List[Tuple] = []
```

Comment:
Instead of a tuple, i'd prefer an AppText or BlfAppText dataclass with timestamp, source and text attributes.